.prettierrc.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
What is the benefit of specifying this as a JSON? Afaik there's no difference in the file format, and Prettier tolerates `.prettierrc` just fine...